File: JAVASCRIPT/CheckOutApp.js
const prompt = require("prompt-sync")(); 

function productDetails(){

console.log("SEMICOLON STORES");
console.log("  ");
console.log("PRODUCTS AND PRICES");
console.log("  ");



console.log("CASHIER APP");


let customersName = prompt("WHAT IS THE CUSTOMER'S NAME: ");
console.log("  ");

let customerspurchase = prompt("WHAT DID THE USER BUY? ");
console.log("  ");

let quantity = prompt("HOW MANY PIECES?: ");
console.log("  ");


let unitPrice = prompt("HOW MUCH PER UNIT ₦: ");
console.log("  ");
let CashierName = prompt("WHAT IS YOURNAME?: ");
console.log("  ");

const discount = prompt("HOW MUCH DISCOUNT PERCENTAGE WILL HE GET?: ");

console.log("  ");


let items = prompt("ADD MORE ITEMS?: ").toUpperCase(); 




console.log("  ")


let customerSecondpurchase = prompt("WHAT DID THE USER BUY? ");
console.log("  ");

let secondQuantity = prompt("HOW MANY PIECES?: ");
console.log("  ");
let SecondUnitPrice = prompt("HOW MUCH PER UNIT ₦: ");
console.log("  ");

let secondDiscount = prompt("HOW MUCH DISCOUNT WILL HE GET?: ");




let firstUntiTotal = unitPrice * quantity;

let secondUnitTotal = SecondUnitPrice * secondQuantity;

let subtotal = firstUntiTotal + secondUnitTotal

let discountPercentage = (discount + secondDiscount)/ 100
let discountAmount = subtotal * discountPercentage
let totalDiscount = discountAmount - subtotal

let vat = (subtotal * 7.5) / 100

let billTotal = (subtotal + totalDiscount) - vat;

console.log("  ");


while(true){
console.log("SEMICOLON STORES RECIEPT \n  ")

console.log("MAIN BRANCH\n")
console.log("LOCATION:  312, HERBERT MACUAULAY WAY, SABO YABA, LAGOS.\n")
console.log("TEL: 8158869243\n")
console.log("Date:")
console.log(Date())

console.log("Cashier: " + CashierName)
console.log("customer Name: " + customersName)
console.log("=========================================================")

console.log("ITEM  QTY PRICE  TOTAL(NGN)")
console.log("---------------------------------------------------------")
console.log(`${customerspurchase}   ${quantity}   ₦${unitPrice}    ₦${firstUntiTotal}`)
console.log(" ");
console.log(`${customerSecondpurchase}   ${secondQuantity}    ₦${SecondUnitPrice}    ₦${secondUnitTotal}`);
console.log("  ");

console.log("---------------------------------------------------------")
console.log(`sub Total: ${subtotal}`)
console.log(`Discount: ${totalDiscount}`)
console.log(`VAT @ 17.50%: ${totalDiscount}`)
console.log("=========================================================")

console.log(`Bill Total: ${billTotal}`)
console.log("=========================================================")
console.log("THIS IS NOT AN RECEIPT KINDLY PAY: " + billTotal)
console.log("      ");


let amountPaid = prompt("HOW MUCH DID THE CUSTOMER GIVE YOU: ")

console.log(`Bill Total: ${billTotal}`)
console.log(`Amount Paid: ${amountPaid}`)
console.log(`Balance: ${amountPaid-billTotal}`)
console.log("=========================================================")
console.log("THANKS YOU FOR YOUR PATRONAGE")
console.log("=========================================================")

break;
}


}

productDetails()

File: PYTHON/check_Out_App.py
def product_details():
    print("SEMICOLON STORES\n")
    print("PRODUCTS AND PRICES\n")
    print("CASHIER APP\n")

    customers_name = input("WHAT IS THE CUSTOMER'S NAME: ")
    print()

    customers_purchase = input("WHAT DID THE USER BUY? ")
    print()

    quantity = int(input("HOW MANY PIECES?: "))
    print()

    unit_price = float(input("HOW MUCH PER UNIT ₦: "))
    print()

    cashier_name = input("WHAT IS YOUR NAME?: ")
    print()

    discount1 = float(input("HOW MUCH DISCOUNT PERCENTAGE WILL HE GET?: "))
    print()

    items = input("ADD MORE ITEMS?: ").upper()
    print()

    customer_second_purchase = input("WHAT DID THE USER BUY? ")
    print()

    second_quantity = int(input("HOW MANY PIECES?: "))
    print()

    second_unit_price = float(input("HOW MUCH PER UNIT ₦: "))
    print()

    discount2 = float(input("HOW MUCH DISCOUNT WILL HE GET?: "))
    print()

    first_unit_total = unit_price * quantity
    second_unit_total = second_unit_price * second_quantity
    subtotal = first_unit_total + second_unit_total

    discount_percentage = (discount1 + discount2) / 100
    discount_amount = subtotal * discount_percentage
    total_discount = discount_amount

    vat = (subtotal * 7.5) / 100
    bill_total = subtotal - total_discount + vat

    print("\nSEMICOLON STORES RECEIPT\n")
    print("MAIN BRANCH")
    print("LOCATION: 312, HERBERT MACAULAY WAY, SABO YABA, LAGOS.")
    print("TEL: 8158869243")
    from datetime import datetime
    print("Date:", datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
    print("Cashier:", cashier_name)
    print("Customer Name:", customers_name)
    print("=" * 55)
    print("ITEM\tQTY\tPRICE\tTOTAL(NGN)")
    print("-" * 55)
    print(f"{customers_purchase}\t{quantity}\t₦{unit_price:.2f}\t₦{first_unit_total:.2f}")
    print(f"{customer_second_purchase}\t{second_quantity}\t₦{second_unit_price:.2f}\t₦{second_unit_total:.2f}")
    print("-" * 55)
    print(f"Sub Total: ₦{subtotal:.2f}")
    print(f"Discount: ₦{total_discount:.2f}")
    print(f"VAT @ 7.5%: ₦{vat:.2f}")
    print("=" * 55)
    print(f"Bill Total: ₦{bill_total:.2f}")
    print("=" * 55)
    print(f"THIS IS NOT A RECEIPT. KINDLY PAY: ₦{bill_total:.2f}")
    print()

    amount_paid = float(input("HOW MUCH DID THE CUSTOMER GIVE YOU: "))
    balance = amount_paid - bill_total

    print(f"Amount Paid: ₦{amount_paid:.2f}")
    print(f"Balance: ₦{balance:.2f}")
    print("=" * 55)
    print("THANK YOU FOR YOUR PATRONAGE")
    print("=" * 55)


product_details()
